app/helpers/subscription_helper.rb

```diff
@@ -0,0 +1,6 @@
+module SubscriptionHelper
+
+  def chargify_subscription_url(subscription)
+    ChargifyService.new(subscription.chargify_subscription_id).subscription_url
```

Comment:
Heh, I think you misunderstood me; what I was thinking was

```
def chargify_subscription_url(subscription)
  return unless Rails.application.secrets.chargify_app_name
  "http://#{Rails.application.secrets.chargify_app_name}.chargify.com/subscriptions/#{subscription.chargify_id}"
end
```

That way we don't have to add those methods into the service.